tools/autograd/derivatives.yaml

```diff
@@ -667,6 +667,17 @@
   self: not_implemented("lstsq")
   A: not_implemented("lstsq")
 
+  # we mark linalg_lstsq non_differentiable wrt to its Tensor inputs because,
```

Comment:
is this comment out of date? It looks like the derivatives for self and b are currently marked as "not implemented"?